src/Common/src/Interop/OSX/System.Security.Cryptography.Native.Apple/Interop.SecKeyRef.cs

```diff
@@ -162,14 +237,29 @@ internal static byte[] GenerateSignature(SafeSecKeyRefHandle privateKey, byte[]
                         out error));
         }
 
+        internal static bool TryGenerateSignature(
+            SafeSecKeyRefHandle privateKey,
+            ReadOnlySpan<byte> source,
+            Span<byte> destination,
+            PAL_HashAlgorithm hashAlgorithm,
+            out int bytesWritten)
+        {
+            Debug.Assert(privateKey != null, "privateKey != null");
+            Debug.Assert(hashAlgorithm != PAL_HashAlgorithm.Unknown, "hashAlgorithm != PAL_HashAlgorithm.Unknown");
+
+            return TryExecuteTransform(
+                source, destination, out bytesWritten,
```

Comment:
Somehow I missed this (and a couple of clones) as having multiple things on this line; I think I read it as another lambda.

Please chop to one per line (for those things which don't fit on one line).